src/ldclient_event_dispatch_httpc.erl

```diff
@@ -27,7 +27,11 @@ send(JsonEvents, PayloadId, Uri, SdkKey) ->
         {"User-Agent", ldclient_config:get_user_agent()},
         {"X-LaunchDarkly-Payload-ID", uuid:uuid_to_string(PayloadId)}
     ],
-    Request = httpc:request(post, {Uri, Headers, "application/json", JsonEvents}, [], []),
+    % Here we don't have information about any specific client instance, so we use the default one
+    Tag = default,
```

Comment:
I am not 100% sure this is entirely correct. Maybe we should make the instance tag part of the behaviour callback, but this will introduce a breaking change.